tests/unit-tests/test-class-lesson.php

```diff
@@ -478,4 +478,36 @@ public function testFirstPrerequisiteIsCorrect() {
 		$first_lesson = Sensei()->lesson::find_first_prerequisite_lesson( $course_with_lessons['lesson_ids'][3], $user_id );
 		$this->assertEquals( $course_with_lessons['lesson_ids'][1], $first_lesson );
 	}
+
+	public function testLessonsAssignedToACourseShouldHavePrerequisitesFromThatCourse() {
```

Comment:
I like to add the [`@covers` annotation](https://phpunit.readthedocs.io/en/9.5/annotations.html#covers) here so that it's clear which function(s) is being tested.